image/templates/helm/stackrox-central/templates/01-central-08-configmap.yaml

```diff
@@ -14,10 +14,10 @@ data:
     {{- ._rox.central._config | nindent 4 }}
     {{- if ._rox.central.enableCentralDB }}
     centralDB:
+      rootDatabaseName: central
```

Comment:
When do you plan to change this config?

Why call it root database? 
It implies the databases have a tree-like structure but I think it does not. How about centralDatabaseName?

Reply:
removed the database name from the config